File: Lesson/iCodeAcademy/lesson_day_23/js/itpec.dart
String grade(int score){
    String ret = '';
    if (score >= 80 && score <= 100){
        ret = 'D';
    } else if(score >= 50 && score < 80){
        ret = 'P';
    } else {
        ret = 'F';
    }
    return ret;
}

void main(){
    print(grade(80)); // D
}

File: Lesson/iCodeAcademy/lesson_day_23/js/itpec.py
# python
def grade(score):
    ret = ''
    if score >= 80 and score <= 100:
        ret = 'D'
    elif score < 80 and score >= 50:
        ret = 'P'
    else:
        ret = 'F'
    return ret

print(grade(80));